python/lookout/sdk/grpc/interceptors/utils.py

```diff
@@ -0,0 +1,130 @@
+from typing import Union
```

Comment:
nit-pick about file name: in general `utils` name better be avoided because it doesn't reflect anything about its content and due to such a general name it tends to grow a lot.

Reply:
I think that it's particularly true for an `utils` module that is present at the root of the project. In that case is like throwing everything you don't know where to put there. In this case IMO the scope is narrower as it regards only gRPC interceptors. But still it can be a black hole of every gRPC interceptor utility. In this particular case maybe something as `base.py` is much more suitable.

Reply:
I'm gonna find another name also for the `lookout.sdk.grpc.utils` module 👍.

Reply:
done